man/man8/zfsprops.8

```diff
@@ -841,6 +841,35 @@ is equivalent to
 .Pc .
 .Pp
 The
+.Sy zstd
+compression using the zstd algorithm ,
+You can specify the
+.Sy zstd
+level by using the value
+.Sy zstd- Ns Em N ,
+where
+.Em N
+is an integer from 1
+.Pq fastest
+to 19
+.Pq best compression ratio .
+.Sy zstd
+You can also specify a negative
+.Sy zstd
+level by using the value
+.Sy zstd- Ns Em N ,
+where
+.Em N
+is an integer from 100
+.Pq fastest
+to 1
+.Pq best compression ratio .
```

Comment:
This is somewhat unclear.
How do we specify a negative value here, zstd--50?
Could you provide an example?

Reply:
That doesn't sound correct at all
the valid values are:
zstd-1 .. zfs-19
zfs-fast-1 (-1), .. zfs-fast-9 (-9)
zstd-fast-10 (-10) .. incremented of 10 .. zstd-fast-100 (-100)
zstd-fast-500 (-500)
zstd-fast-1000 (-1000)